src/main/java/jenkins/scm/api/SCMEvent.java

```diff
@@ -256,6 +257,17 @@ public String getOrigin() {
     }
 
     /**
+     * Return a description of the event.
+     *
+     * @return the description or {@code null} if no description can be provided.
+     * @since TODO
+     */
+    @CheckForNull
+    public String description() {
```

Comment:
Without verb before it very unclear what this method do, either getting, either setting, either reseting